### ISSUE TITLE
Python 3.8 DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats

### ISSUE THREAD

**Issue by fr-ser** [confluentinc/confluent-kafka-python#763]
Description
===========

Producing something generates this deprecation warning in Python 3.8.1:
`DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats`

How to reproduce
================
Minimal reproducible repo here: https://github.com/fr-ser/confluent-kafka-python-issue

Code:
```python
import confluent_kafka

p = confluent_kafka.Producer({"bootstrap.servers": "PLAINTEXT://localhost:9092"})
p.produce("test-topic", b"my_value", b"my_key")
```

Checklist
=========
Please provide the following information:

 - [x] confluent-kafka-python and librdkafka version (`confluent_kafka.version()` and `confluent_kafka.libversion()`): version ('1.3.0', 16973824) - libversion ('1.3.0', 16974079)
 - [x] Apache Kafka broker version: 2.12-2.4.0
 - [x] Client configuration: n/a
 - [x] Operating system: Ubuntu 19.10
 - [x] Provide client logs (with `'debug': '..'` as necessary): n/a
 - [x] Provide broker log excerpts: n/a
 - [x] Critical issue: no, just a warning

**Comment by rnpridgeon** [comment]
Thanks for reporting. This is something that needs to be addressed for our Windows binary wheel builds as well.

**Comment by mbogner** [comment]
```
DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats
 producer.produce(topic=topic, value=json_message.encode('utf-8'))
```

**Comment by alena-m** [comment]
I got the same message from producer: **DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats**

```
bootstrap_servers: b-1.xxx.amazonaws.com:9094,b-2.xxx.amazonaws.com:9094 (AWS Managed Kafka)
Python 3.8.6
confluent-kafka==1.5.0
Debian GNU/Linux 10 (buster)
```

**Comment by edenhill** [comment]
This is fixed in the upcoming v1.5.2 release.

**Comment by woomoth** [comment]
What is the latest version that does not have this issue (so my project can go forward before 1.5.2 is released)?  Thanks!

**Comment by aoqfonseca** [comment]
When this release will come out?

**Comment by mbogner** [comment]
1.5.0 from july 2020 is still the latest release. is there any release planned for the near future?

having a confirmed bug open for more than a year doesn't really raise my confidence level in using a library tbh.

**Comment by NormanLYJ** [comment]
I was evaluating Confluent managed kafka and the same error led me to this page, this kind-of blocked my progress, if it has not been fixed yet, is there a workaround at least?
--talking abt Python implementation

**Comment by mbogner** [comment]
Only workaround is to silence the message

**Comment by edenhill** [comment]
v1.6.0 will be released within a couple of weeks

**Comment by luckysunil** [comment]
Thanks @edenhill   for the update.

**Comment by pranavrth** [comment]
This was fixed in v1.6.0

Closing the issue.
